insteon_mqtt/device/base/Base.py

```diff
@@ -649,6 +653,23 @@ def get_model(self, on_done=None):
                                                    on_done)
         self.send(msg, msg_handler)
 
+    def _set_operating_flag_msg(self, cmd2):
+        """Creates a set operating flags message
+
+        This will create the appropriate message based on the device's
+        engine (extended for i2cs, else standard)
+
+        Args:
+          cmd2 (int): the set operating flags option (i.e. 0x04 to enable
+                      resume dim)
+        """
+        cmd1 = Msg.CmdType.SET_OPERATING_FLAGS
+        if self.db.engine == 2:
+            return Msg.OutExtended.direct(self.addr, cmd1, cmd2,
+                                            bytes([0x00] * 14), crc_type='D14')
+        else:
+            return Msg.OutStandard.direct(self.addr, cmd1, cmd2)
```

Comment:
It probably won't matter since we are going to add `get_engine` to `refresh` but given that all devices made in the last 5+ years are i2cs, we should default to that and not the old standard length message.